dashboard/views.py

```diff
@@ -467,19 +467,20 @@ def grade_distribution(request, course_id=0):
 
     df.sort_values(by=['current_grade'], inplace=True)
     df.reset_index(drop=True, inplace=True)
+    if df[df['current_grade'] > 100.0].shape[0] > 0:
+        summary['graph_upper_limit'] = int((5 * round(float(df['current_grade'].max()) / 5) + 5))
```

Comment:
Because the FE ticks are like 5 ,10, 20, 25 so I calculate the highest tick could be based on the highest grade Max. I need to dig back on this since it's been 2 years